packages/tokamak/contracts/contracts/L1/token/TON.sol

```diff
@@ -0,0 +1,21 @@
+// SPDX-License-Identifier: MIT
```

Comment:
I think we can assume there is TON contract in L1. If you use this only for tesing and will be removed in production. It's okay for me! 🙇🏼‍♂️